From b96b0fc07c49724f252a5cd35f2d028f91529068 Mon Sep 17 00:00:00 2001
From: Jeena <hello@jeena.net>
Date: Tue, 15 Aug 2017 16:53:05 +0200
Subject: [PATCH] Drop unnecessary format-string.

Code got refactored in 3.25/master but it boils down to dropping this
extra parameter.
---
 src/contacts-contact-pane.vala | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/contacts-contact-pane.vala b/src/contacts-contact-pane.vala
index 8e100f2..fce832b 100644
--- a/src/contacts-contact-pane.vala
+++ b/src/contacts-contact-pane.vala
@@ -451,7 +451,6 @@ public class Contacts.ContactPane : Notebook {
 					 DialogFlags.MODAL,
 					 MessageType.ERROR,
 					 ButtonsType.OK,
-					 "%s",
 					 _("You need to enter some data"));
       dialog.show ();
       dialog.response.connect ( () => {
@@ -463,7 +462,6 @@ public class Contacts.ContactPane : Notebook {
 					 DialogFlags.MODAL,
 					 MessageType.ERROR,
 					 ButtonsType.OK,
-					 "%s",
 					 _("No primary addressbook configured"));
       dialog.show ();
       dialog.response.connect ( () => {
@@ -482,7 +480,6 @@ public class Contacts.ContactPane : Notebook {
 					DialogFlags.MODAL,
 					MessageType.ERROR,
 					ButtonsType.OK,
-					"%s",
 					_("Unable to create new contacts: %s"), e.message);
 	  }
 
@@ -493,7 +490,6 @@ public class Contacts.ContactPane : Notebook {
 					DialogFlags.MODAL,
 					MessageType.ERROR,
 					ButtonsType.OK,
-					"%s",
 					_("Unable to find newly created contact"));
 	  }
 
-- 
2.14.1

